### DISCUSSION TITLE
After update - overview pic misaligned but reference pic aligned

### DISCUSSION THREAD

**Discussion by danieldownload** [jomjol/AI-on-the-edge-device#3072] [General]
Hi there,

just updated my two water meters to the lates release coming from 15.3. One of the meters works normal after the update, the other one shows alignment issues.
The overview pictures is slightly rotated but the reference image is correct.

What can I do?

<img width="1029" alt="Bildschirmfoto 2024-05-09 um 10 46 52" src="https://github.com/jomjol/AI-on-the-edge-device/assets/26095164/d98c0d57-b54c-4565-bca5-c264f4d6727f">

<img width="1032" alt="Bildschirmfoto 2024-05-09 um 10 47 05" src="https://github.com/jomjol/AI-on-the-edge-device/assets/26095164/8a20c84c-4c86-4aef-a22c-ab8bb2a1dada">

**Comment by SybexX** [comment]
This happens when the system does not recognize the alignment marks correctly and therefore misaligns the image.
You have much better picture quality below, why don't you put it there?
![329186642-d98c0d57-b54c-4565-bca5-c264f4d6727f](https://github.com/jomjol/AI-on-the-edge-device/assets/587201/12041905-798a-4c1e-bf74-0c36a7557a67)
If you use the current rolling, you can use the extended zoom function, this allows you to further improve the evaluation.
https://github.com/jomjol/AI-on-the-edge-device/actions/runs/8992820658
